tools/llvm-slicer.cpp

```diff
@@ -958,7 +958,7 @@ static uint32_t parseAnnotationOpt(const std::string& opt)
 
 int main(int argc, char *argv[])
 {
-    llvm::sys::PrintStackTraceOnErrorSignal();
+    llvm::sys::PrintStackTraceOnErrorSignal(llvm::StringRef());
```

Comment:
This needs compilation guards, since in LLVM 3.8 the signature is different (there it takes a bool optionally)